main.py

```diff
@@ -286,6 +319,44 @@ def validateSettings(staticLetters) -> bool:
     return True
 
 
+def normalized(sett: dict) -> list:
+    total = sum(sett[i] for i in sett)
+    return [(sett[i]/total, i) for i in sett]
+
+
+def orderLetters(configs: tuple = BIGRAMS_CONFIGS) -> str:
+
+    # Generate monograph sets for each language
+    monoSets = []
+    for config in configs:
+        if config.weight <= 0:
+            continue
+        with open(config.path, 'r') as biFile:
+            bis = {line[:2]: float(line[3:]) for line in biFile}
+        notNorm = {}
+        for bi in bis:
+            for char in bi:
+                if char in AUTO_LAYER_IGNORE:
+                    continue
+                if char in notNorm:
+                    notNorm[char] += bis[bi]*config.weight
```

Comment:
I might be wrong, but are you sure what it is correct to **first** multiply the frequencies by some weight and **then** normalize the frequencies?
To me, it sounds like normalizing would remove the weighting.

Reply:
You are absolutely write, I don't know how I let that mistake slip through, thanks!
Done in 444e09cfcf29a2dc6c1ad61332d4f2e98f975ae6